core/client/app/components/gh-profile-image.js

```diff
@@ -28,12 +29,25 @@ export default Ember.Component.extend({
         return `background-image: url(${url})`.htmlSafe();
     }),
 
-    imageBackground: Ember.computed('email', 'size', function () {
-        var email = this.get('email'),
+    trySetValidEmail: function () {
+        var email = this.get('email');
+        if (validator.isEmail(email)) {
+            this.set('validEmail', email);
+        } else {
+            this.set('validEmail', '');
+        }
+    },
+
+    onEmailChange: Ember.observer('email', function () {
```

Comment:
Didn't know this hook even existed, my lack of no Ember usage outside some occasional Ghost contribution shows off :sweat_smile:  This is definitely a better way to go with observing changes. Thanks for a tip!